ui/src/app/features/collection-manager-subject-right/collection-manager-subject-right/collection-manager-subject-right.component.scss

```diff
@@ -52,3 +51,8 @@
     }
   }
 }
+
+.image-holder-collection-container {
+	display: flex;
```

Comment:
Do you really need flex here?

Reply:
it aligns loading spinner in the collection container

Reply:
so it causes .image-holder-collection to contract (you have your grid not for a full width of its container). This behavior is observed when you have broken image. So as a fix for this you can set width: 100% for its host container (ImageHolderCollectionComponent).